algorithms/ar-cochprb/java/coin_change_problem.java

```diff
@@ -0,0 +1,51 @@
+class coinchange{ 
+      
+    // Returns the count of ways we can  
+    // sum S[0...m-1] coins to get sum n 
+   static long countWays(int S[], int m, int n) 
```

Comment:
Codacy found an issue: [Use explicit scoping instead of the default package private level](https://app.codacy.com/gh/FOSS-UCSC/FOSSALGO/pullRequest?prid=6359243)